.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I believe we need an extra config for `yapf` check and maybe other "heavy" ones. e.g. `.pre-commit-config-heavy.yaml` and an additional tox env for heavy checks.

Reply:
Well... arguably, we could do anything in a one linee of `bash -c "..."` but this would not make it easy to use. If the user needs to add extra parameters (especiall long ones) is likely for use to make the UX worse than better.

Just to clarify: I am not againt the idea, just that I do not see many benefits. The good part is that we can easily postpone the yapf work for another change.

Reply:
I have a strong feeling that you didn't get the idea... I'm talking about a separate tox env with `pre-commit --config heavy.yaml`